src/client/content_mapblock.cpp

```diff
@@ -969,35 +969,44 @@ void MapblockMeshGenerator::drawPlantlike()
 		break;
 	}
 
+	int o, q;
+	if ((p.Y & 1) == 0) {
+		o = 1;
+		q = 359;
```

Comment:
`-1` works just fine here. This means that `q` can be substituted by `-o` in the changes below.